src/lib/args-manager.ts

```diff
@@ -54,6 +54,7 @@ export function extractArgs(inputArgs: string[]) {
   const noColors = extractCommand(args, '--noColors');
   const noClear = extractCommand(args, '--noClear');
   const silent = extractCommand(args, '--silent');
+  const signalEmittedFiles = extractCommand(args, '--signalEmittedFiles') && !args.includes('--listEmittedFiles');
```

Comment:
I think that we have some naming problem here...
maybe this `const` should be called `dontPrintTSFILE`?

Reply:
@gilamran More names, more complexity, if `--signalEmittedFiles` is specified, I'd rather keep `signalEmittedFiles`
`dontPrintTSFILE` looks surprising for a parser, no?
The role of `&& !args.includes('--listEmittedFiles');` is to virtually make as if `--signalEmittedFiles` was not specified.

Reply:
It's a bit confusing that we do send signals for emitted file if `--listEmittedFiles` is specificed. and `--signalEmittedFiles` meaning is actually "Only signal emitted file, don't print them".
If you can think of a better name, that would be great. if not that's also ok...

Reply:
@gilamran I tried with your wording, but I'm really annoyed when if comes to this in `tsc-watch.ts` for the dual usage (both call tsc with right args, and call print) :confused: 
```typescript
  if (dontPrintTSFILE) {
    nodeArgs.push(`--listEmittedFiles`);
  }
```

The variable could be named `listEmittedFilesOrHideTSFILE` to match all needs
but hiding this complexity in the parser will potentially make this more maintainable.

I think I will stop here, however thank you for the suggestion

Reply:
I understand, what about we'll have 2 variables one that extract the `signalEmittedFiles` and one that indicates if `--listEmittedFiles` was requested. Lets call the 2nd one `requestedToListEmittedFiles`.
In this case we can do the `if` like this:
```
if (signalEmittedFiles && !requestedToListEmittedFiles) {
  return;
}
```